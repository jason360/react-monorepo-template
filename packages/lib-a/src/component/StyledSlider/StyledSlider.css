.MuiSlider-root {
  color: #20b2aa;
}

.MuiSlider-root:hover {
  color: #2e8b57;
}