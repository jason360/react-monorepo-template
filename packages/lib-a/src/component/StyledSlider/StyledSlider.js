import * as React from 'react';
import Slider from '@mui/material/Slider';
import './StyledSlider.css';

const StyledSlider = (props) => {
  return <Slider {...props} />;
}

export default StyledSlider;