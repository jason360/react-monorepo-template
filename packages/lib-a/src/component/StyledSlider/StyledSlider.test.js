import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import StyledSlider from './StyledSlider';

describe('<StyledSlider />', () => {
    test('renders StyledSlider correctly', () => {
        render(<StyledSlider size="small" />);
        expect(screen.getByRole('slider')).toMatchSnapshot();
    });
});