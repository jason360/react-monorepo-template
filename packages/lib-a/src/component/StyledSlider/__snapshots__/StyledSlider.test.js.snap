// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StyledSlider /> renders StyledSlider correctly 1`] = `
<input
  aria-orientation="horizontal"
  aria-valuemax="100"
  aria-valuemin="0"
  aria-valuenow="0"
  data-index="0"
  style="border: 0px; height: 100%; margin: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 100%; direction: ltr;"
  type="range"
  value="0"
/>
`;
